04_setup_ironic.sh

```diff
@@ -122,7 +122,7 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
         _tmpfiles="$_tmpfiles $OLM_DIR"
 
         pushd $OLM_DIR
-        curl -O https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/${VERSION}.0/opm-linux.tar.gz
+        curl -O https://mirror.openshift.com/pub/openshift-v4/$(uname -m)/clients/ocp/${VERSION}.0/opm-linux.tar.gz
```

Comment:
Why do you need to change this file? The agent workflow uses a (slightly) different one: https://github.com/openshift-metal3/dev-scripts/blob/master/agent/04_agent_prepare_release.sh

Reply:
yeah, this is just multi-arch clean up. Probably belongs in another PR, will remove. 👍